stdlib/contrib/RohanSreerama5/naiveBayesClassifier/zoo_data.csv

```diff
@@ -0,0 +1,102 @@
+Animal_name,hair,feathers,eggs,milk,airborne,aquatic,predator,toothed,backbone,breathes,venomous,fins,legs,tail,domestic,catsize,type
```

Comment:
Would it be possible to create a test case out of this data? That way others can see how to use the classifier with pure Flux code? 

To create a test case create a `bayes_test.flux` file (call it whatever you think is best it just needs to end in `_test.flux`) and call the `naiveBayes` function from within that test code. You can embed the csv directly as a string in that file. See the other _test.flux files there are lots of examples to follow.

Reply:
If you don't know what the `outData` might look like for your test, you can try it out in the Flux repl.